integration/entity.go

```diff
@@ -54,6 +65,11 @@ func newEntity(entityName, entityType string, storer persist.Storer) (*Entity, e
 	return &d, nil
 }
 
+// IsDefaultEntity returns true if entity is the default one (has no identifier: name & type)
```

Comment:
Do we know how the agent behaves with an entity without name nor type?

Reply:
This public function is only used to get the default entity, I would maybe remove it or make it private. (Look to following comments for an alternative proposal).

Reply:
We should change the whole `Entitites` from list to a map as commented https://github.com/newrelic/infra-integrations-sdk/pull/71/files#diff-34fc2068cceaa93f8d6b312c44397541R53

By doing so we don't need to treat the default-entity in a separate way.

Reply:
Regarding the agent, good point:

We talked on this and the agent is supposed to treat that entry as no-entity case.
Btw I'll test this case and will come back here to confirm.

Reply:
Privatized method